api/backend/lock.go

```diff
@@ -2,6 +2,8 @@ package backend
 
 import "sync"
 
+// TODO: dave-gray101 8/11: I think this is unused. Check back in a while to see if this should be removed, or refactored to _be_ used.
```

Comment:
correct, let's get rid of it as it was just used by some backends with the shared lock (that we are removing from here)